### PR TITLE
Allow kubectl from localhost

### PR DESCRIPTION
**Pull request by gouthampacha**

### CODE REVIEW

File: roles/master/tasks/main.yml [hunk 1]
```diff
@@ -1,14 +1,27 @@
 - name: initialize kubeadm on master
-  command: kubeadm init --pod-network-cidr=10.244.0.0/16 --token={{ kubernetes_token }} --apiserver-advertise-address={{ ansible_default_ipv4.address }}
+  shell: |
```
Comment (tombarron) [inline roles/master/tasks/main.yml:2]: Any particular reason for switching from command module to shell?  The former is generally preferred unless one needs to use shell redirection, pipes, etc.

**Review by tombarron — APPROVED** [review]
Awesome, thanks!

File: roles/master/tasks/main.yml (hunk 1 above)
Comment (gouthampacha) [reply roles/master/tasks/main.yml:2]: I couldn't break command into multiple lines, with either of these forms:

```
 command: >-
   long-multi
   line-command
```


or,


```
   command: |
    long-multi
    line command
```

Comment (tombarron) [reply roles/master/tasks/main.yml:2]: Make sense, and thanks for breaking into multiple lines!

**Pull request merged**